studio/plugins/sketch/index.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs')
+const path = require('path')
+const { exec } = require('child_process')
+const { setup, sendRequest, sendNotification, RPCError } = require('stdio-jsonrpc')
+const { createNewSketchFile, writeSketchFile, generateId } = require('sketch-file');
+
+function modifySketchTemplate(layers, textStyles, output) {
+  const sketchDoc = createNewSketchFile(generateId(output));
+
+  sketchDoc.document.layerTextStyles.objects = textStyles;
+  sketchDoc.pages[0].layers = sketchDoc.pages[0].layers.concat(layers);
+
+  return writeSketchFile(sketchDoc, output);
+}
+
+Promise.all([
+  sendRequest('workspacePath'),
+  sendRequest('compilerPath')
+])
+  .then(([workspace, compiler]) => {
+    if (!workspace) {
+      throw new Error('missing workspace path')
+    }
+
+    if (!compiler) {
+      compiler = require.resolve('lonac')
+    }
+
+    // TODO: change that
+    const output = path.join(__dirname, './generated')
+
+    try {
+      fs.mkdirSync(output, {recursive: true})
+    } catch (err) {
+      if (err.code !== 'EEXIST') {
+        throw err
+      }
+    }
+
+    return new Promise((resolve, reject) => {
+      exec(`node "${compiler}" workspace js "${workspace}" "${output}" --framework=reactsketchapp`, (err, stdout, stderr) => {
```

Comment:
I know we don't have a proper place to put the Sketch file yet, but should we put the generated JS files somewhere in `os.tmpdir()`?